samples/modules/lvgl/demos/sample.yaml

```diff
@@ -4,13 +4,22 @@ sample:
 common:
   modules:
     - lvgl
-  harness: none
+  harness: console
   filter: dt_chosen_enabled("zephyr,display")
+  extra_args:
+    - platform:mimxrt1060_evk:SHIELD=rk043fn66hs_ctg
+    - platform:mimxrt1170_evk/mimxrt1176/cm7:SHIELD=rk055hdmipi4ma0
+    - platform:mimxrt595_evk/mimxrt595s/cm33:SHIELD=rk055hdmipi4ma0
   tags:
     - samples
     - display
     - lvgl
     - gui
+  harness_config:
+    fixture: fixture_display
+    type: one_line
+    regex:
+      - "\\w+ free bytes, \\w+ allocated bytes, overhead = \\w+ bytes"
```

Comment:
it feels fragile to depend on a string coming from LVGL? maybe just use the same approach as other sample, i.e log something of your own?

Reply:
The heap stat is only available with `LV_Z_MEM_POOL_SYS_HEAP` option, using the malloc heap this will fail.

Reply:
Thanks @faxe1008 , I update this check to compare both, please review